database/config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  user: 'root',
+  password: '',
+  database: 'revservice'
```

Comment:
in real scenario you would not want to expose your creds:
 1) put this in a `.env` file, use `dotenv` npm package and access via `process.env.DB_USER` or something. add `.env` to `.gitignore`
 2) add this file to `.gitignore` and instead put a placeholder file here called `database\config_example.js` with [USER_NAME_HERE] etc.. then make a note in README